app/js/skrollr.js

```diff
@@ -18,6 +18,12 @@ angular.module("sn.skrollr", [])
 
     this.config = {};
 
+    this.skrollrInstance = {};
+
+    this.hasBeenInitialised = false;
+
+    this.serviceMethods = {};
+
     this.$get = [
```

Comment:
@jamesjwarren just realised these variables should be commented as well, a task to do later